cmd/tengo/main.go

```diff
@@ -55,10 +55,34 @@ func main() {
 	inputData, err := ioutil.ReadFile(inputFile)
 	if err != nil {
 		_, _ = fmt.Fprintf(os.Stderr,
-			"Error reading input file: %s", err.Error())
+			"Error reading input file: %s\n", err.Error())
 		os.Exit(1)
 	}
 
+	if compileOutput != "" {
+		if compileOutput, err = filepath.Abs(compileOutput); err != nil {
+			fmt.Fprintf(os.Stderr,
+				"Error file path: %s\n", err.Error())
+			os.Exit(1)
+		}
+	}
+
+	inputFile, err = filepath.Abs(inputFile)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error file path: %s\n", err)
+		os.Exit(1)
+	}
+
+	if err := os.Chdir(filepath.Dir(inputFile)); err != nil {
```

Comment:
I think we shouldn't change the working directory to the script file. Import script files using a relative path to the current script file makes sense, but the actual working directory should be the same directory that the user executes the code (either Tengo CLI or an inline Tengo code in Go). That way, if the script calls `os` package functions (e.g. `os.read_file`), they will use the current working directory, not the script file directory. 

In other words, the import path for each module should be used only for importing other modules, but nothing else.

Reply:
OK I will not mess with os.Chdir :smile: